src/utilities/test-utils.ts

```diff
@@ -46,6 +62,7 @@ export const TestUtils = {
     randomKey: _randomKey,
     randomValue: _randomValue,
     randomWord: _randomWord,
+    randomObject: _randomObject,
```

Comment:
Nitpick - would you mind alphabetizing this property in this object?

Reply:
Done